SEPADirectDebit/src/main/java/com/braintreepayments/api/SEPADirectDebitBrowserSwitchRequest.kt

```diff
@@ -0,0 +1,22 @@
+package com.braintreepayments.api
+
+import androidx.annotation.RestrictTo
+
+/**
+ * Request parameters associated with a [SEPADirectDebitPendingRequest.Started]
+ */
+class SEPADirectDebitBrowserSwitchRequest internal constructor(
+    browserSwitchPendingRequest: BrowserSwitchPendingRequest.Started
+) {
+
+    private var browserSwitchPendingRequest: BrowserSwitchPendingRequest.Started
+
+    init {
+        this.browserSwitchPendingRequest = browserSwitchPendingRequest
+    }
+
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
just for my knowledge, what does this scope setting do?

Reply:
This is our Kotlin equivalent of Java's package-private - so essentially saying this method is "internal" to our SDK an not covered by semantic versioning